example/lib/constants/configs.dart

```diff
@@ -3,10 +3,10 @@ import 'package:phyllo_connect/phyllo_connect.dart';
 class Configs {
   Configs._();
 
-  static const String clientId = '<client id here>';
+  static const String clientId = '2778c997-f4bc-401d-95c4-6f2178b85404';
   
-  static const String clientSecret = '<client secret here>';
+  static const String clientSecret = '535b916c-292b-4771-bd6a-41dc4a53db6f';
```

Comment:
@erpankajpatel , we can remove the secret & id from here

Reply:
@chiragphyllo updated